pkg/kubelet/dockershim/docker_service.go

```diff
@@ -314,6 +324,9 @@ type dockerService struct {
 	// the docker daemon every time we need to do such checks.
 	versionCache *cache.ObjectCache
 
+	// caches the info from the runtime.
+	infoCache *cache.ObjectCache
```

Comment:
can we just keep the `dockertypes.Info` object? What's the benefit of using the cache with a single value in it?

Reply:
oh, I see, the expiration in case it was restarted